test/pytest/test_mbox_multi_system.py

```diff
@@ -0,0 +1,22 @@
+import serial
+import subprocess
+import pytest
+from pexpect.fdpexpect import fdspawn
+
+@pytest.mark.timeout(400)
+@pytest.mark.parametrize('core_num', range(8))
+
+def test_rtps_hpps(qemu_instance_per_fcn, host, core_num):
```

Comment:
This test seems to reset QEMU after each core is tested.  Does it need to do that?  It adds extra time to the test, so if it's not needed, please see if it can be corrected.